packages/cid-verifier/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "cid-verifier",
+  "version": "1.0.0",
+  "description": "Verify that a CID's content is safe for the web",
+  "private": true,
+  "type": "module",
+  "main": "./dist/worker.mjs",
```

Comment:
See this https://github.com/web3-storage/reads/pull/47 . We recently discovered this was a problem, and I fixed for edge-gateway. We need the changes of the PR propagated too here